docs/iot/guides/things-api-guide-5.0.md

```diff
@@ -0,0 +1,443 @@
+# API Usage
+
+SmartThings Thing API is a service that provides connection to SmartThings Cloud. Also, it acts as an interface for resource applications to be controlled by the SmartThings applications.
+
+**Figure: SmartThings Thing overall architecture**
+
+![SmartThings Thing overall architecture](./media/smartthings_thing_overall_architecture.png)
+
+The main SmartThings Thing API features include:
+
+- Master API: Connecting to SmartThings Cloud with Samsung account information.
+- Resource API: Interfaces for Resource applications to be controlled by SmartThings application.
+
+> **Note**
+>
+> You can use both the APIs separately, for each Master and Resource application or together for the hybrid (master+resource) applications.
+
+## Prerequisites
+
+You must connect to the Internet.
+
+## Registering Samsung account
+
+You must have a [Samsung account](https://account.samsung.com/membership/index.do) and access to the [Developer Workspace](https://devworkspace.developer.samsung.com).
+
+> **Note**
+>
+> Log on to the [Developer Workspace](https://devworkspace.developer.samsung.com) and click Development > SmartThings Device > Cloud-connected > My Cloud-connected devices > Create.
+
+## Configuring SmartThings API
+
+Update the following pages on the SmartThings cloud:
+ - Device Info
+ - Self-publish
+
+> **Note:**
+>
+> For some fields, the values entered in the pages must match with the values specified in your JSON file.
+
+  **Table: Device information - page fields matching with Master.json**
+
+  Field|master.json
+  :---:|----
+  Device type|deviceType
+  Device name|deviceName
+  MNID|mnid
+  VID|vid
+  Device onboarding ID|setupId
+
+  **Table: Self-publish - page fields matching with Resource.json**
+
+  Field|resource.json
+  :---:|----
+  Capabilities|resources > uri
+  Resources|resourceTypes > type
+
+**Figure: SmartThings Thing device information**
+
+![SmartThings Thing Device info](./media/smartthings_thing_device_info.png)
+
+**Figure: SmartThings Thing add capabilities**
+
+![SmartThings Thing Add capabilities](./media/smartthings_thing_add_capabilities.png)
+
+**Figure: SmartThings Thing self publish**
```

Comment:
To maintain consistency, with all other Tizen docs, please give the figure caption below the figure.

Reply:
Ok. Done.